src/ast.rs

```diff
@@ -217,24 +217,140 @@ impl Identifier {
         Identifier { values }
     }
 
-    /// Check if an identifier is relative
+    /// Check if an identifier is canonical
     ///
-    /// An identifier is considered as a relative one if it starts with `this` or `super`
-    /// keyword.
-    pub fn is_relative(&self) -> bool {
+    /// An identifier is considered as canonical if all relative identifier values
+    /// (`IdentifierValue::This`, `IdentifierValue::Super`) are not present.
```

Comment:
English nitpick - `all [] are not present` -> `none of [] are present`

Reply:
Fixed.